### PR TITLE
Add some javascript unit tests to momentUtils.js

### PR DESCRIPTION
**Pull request by nicmarti**
Started to add simple tests to momentUtil.js

### CODE REVIEW

**Comment by nicmarti** [comment]
@GuillaumeDavy can you have a look at my code and maybe see if you want to add more tests?

File: frontend/src/utils/momentUtils.js [hunk 1]
```diff
@@ -23,6 +23,22 @@ import _ from 'lodash';
...
+  if(!end){
+    return "invalid endDate";
+  }
+
+  if(start.isAfter(end)){
+    return 'invalid range, endDate must be after startDate';
+  }
+
+  if(start.isSame(end)){
+    return `${start.format(panelFormatWithYear)}`;
```
Comment (tkuy) [inline frontend/src/utils/momentUtils.js]: `return start.format(panelFormatWithYear);` is enough in this case.

File: frontend/src/utils/momentUtils.test.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {renderRangeWithYear, renderRange, weekRangeOfDate} from './momentUtils';
+import moment from 'moment';
+
+// eslint-disable-next-line
+test('renderRangeWithYear a range year with different month', () => {
+    const startDate = moment('2020-01-22');
```
Comment (tkuy) [inline frontend/src/utils/momentUtils.test.js]: Should use `moment.utc('2020-01-22')` instead according to our choice to use moment in utc only.

Comment (nicmarti) [reply frontend/src/utils/momentUtils.test.js]: agree

File: frontend/src/utils/momentUtils.js (hunk 1 above)
Comment (nicmarti) [reply frontend/src/utils/momentUtils.js]: ok done

**Pull request merged**